src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,23 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
The class implementation should not begin with an empty line. Please remove the unnecessary empty lines at the beginning of the class definition.